### PR TITLE
Added support for custom structure in bar charts

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?
Added support for custom structure in bar charts so we can customize bar order & label

## How it looks?

Screen Shot 2021-01-18 at 21 49 47

### CODE REVIEW

**Review by dvaldivia — APPROVED** [review]

File: portal-ui/src/screens/Console/Dashboard/Prometheus/utils.ts [hunk 1]
```diff
@@ -523,20 +550,44 @@ export const getWidgetsWithValue = (payload: any[]) => {
...
-            return { name: metricName, a: parseInt(lastValue[1]) };
-          });
+              const elements = get(metricTake, "values", []);
+
+              const sortResult = elements.sort(sortFunction);
+              const lastValue = sortResult[sortResult.length - 1];
+
+              return {
+                name: structureItem.displayTag,
+                a: parseInt(lastValue[1]),
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Dashboard/Prometheus/utils.ts:574]: should we add a validaiton here to prevent npe?

Comment (Alevsk) [inline portal-ui/src/screens/Console/Dashboard/Prometheus/utils.ts:574]: should we add a validation here to prevent `npe`?

**Review by Alevsk — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]
Thanks for building assets :cake:

**Review by cesnietor — APPROVED** [review]
Tested LGTM

**Pull request merged**